.about.yml

```diff
@@ -0,0 +1,109 @@
+---
+# .about.yml project metadata
+#
+# Short name that acts as the project identifier (required)
+name: C2
+
+# Full proper name of the project (required)
+full_name: C2
+
+# The type of content in the repo
+# values: app, docs, policy
+type: app
+
+# Describes whether a project team, working group/guild, etc. owns the repo (required)
+# values: guild, working-group, project
+owner_type: project
+
+# Name of the main project repo if this is a sub-repo; name of the working group/guild repo if this is a working group/guild subproject
+parent: C2
+
+# Maturity stage of the project (required)
+# values: discovery, alpha, beta, live
+stage: live
+
+# Whether or not the project is actively maintained (required)
+# values: active, deprecated
+status: active
+
+# Description of the project
+description: an approval process automation tool
+
+# Should be 'true' if the project has a continuous build (required)
+# values: true, false
+testable: true
+
+# Team members contributing to the project (required)
+# Items:
+# - github: GitHub user name
+#   id: Internal team identifier/user name
+#   role: Team member's role; leads should be designated as 'lead'
+team:
+- github: afeld
+  id: afeld
+  role: developer
+- github: annalee
+  id: annalee
+  role: developer
+- github: anthonygarvin
+  id: tony
+  role: developer
+- github: phirefly
+  id: raphy
+  role: lead
+- github: pkarman
+  id: pkarman
+  role: developer
+- github: yozlet
+  id: yoz
+  role: developer
+
+# Partners for whom the project is developed
+partners:
+- GSA National Capital Region
+- 18F
+- GSA Common Acquisition Platform
+
+# Brief descriptions of significant project developments
+milestones:
+- 
+
+# Technologies used to build the project
+stack:
+- Rails
```

Comment:
May also want to throw `- Ruby` in here, and any other relevant technologies you can think of.